source/fab/steps/compile_fortran.py

```diff
@@ -218,8 +214,6 @@ def process_file(self, analysed_file: AnalysedFile):
             Before compiling a file, we calculate the combo hashes and see if the output files already exists.
 
         """
-        # todo: include compiler version in hashes
```

Comment:
Has this been done? `_get_obj_combo_hash` doesn't seem to do it. Is it no longer relevant?

Reply:
Yes, done and merged in #150. It's on line 267, in `_get_obj_combo_hash`.

Reply:
(now on line 145, post merge)